style.css

```diff
@@ -1,4 +1,4 @@
-h1 {
-  color: rgb(102, 51, 153);
-  text-align: center;
-}

```

Comment:
- [ ] The previous reviewer requested that you update your README with the author's name and clickable links to your professional & social media pages for easy contact. We are making this request now so you know that it is standard for all your microverse projects going forward. 

**_Example_** - > 
> To be able to get a clickable link use this markdown syntax - 
`[GitHub handle](https://github.com/yash244466666)` and it will appear as shown below: [Read about creating links with markdown syntax here](https://anvilproject.org/guides/content/creating-links)

## Author
### Your name here
[GitHub handle](https://github.com/yash244466666)
[Twitter handle](https://github.com/yash244466666)
[LinkedIn handle](https://github.com/yash244466666)